content/doc/administration/requirements/java.adoc

```diff
@@ -9,11 +9,12 @@ There are separate run and job execution requirements for Jenkins installations.
 
 Modern Jenkins versions have the following Java requirements:
 
-* Java 8 is the **ONLY** currently supported runtime environment, both 32-bit and 64-bit versions are supported
+* Java 8 runtime environments, both 32-bit and 64-bit versions are supported
+* Since Jenkins `2.163` footnote:[since `2.155` but it was made easier in `2.163`], Java 11 runtime environments is supported
+** in order to run Jenkins on Java 11, there are few precautions to take, which are detailed link:/doc/administration/requirements/jenkins-on-java-11[here].
```

Comment:
To improve accessibility it's better to avoid _click here_: https://www.w3.org/QA/Tips/noClickHere

Reply:
is it better with this @MRamonLeon ?

```suggestion
** in order to run Jenkins on Java 11, there are few precautions to take, which are detailed in the link:/doc/administration/requirements/jenkins-on-java-11[Running Jenkins on Java 11] documentation.
```